pages/CheckingPage.js

```diff
@@ -0,0 +1,66 @@
+import BasePage from '../pages/BasePage';
+
+class CheckingPage extends BasePage {
```

Comment:
This class mixes elements from the checking account creation page and the one about viewing existing accounts. Make sure to split this into two classes and allocate elements accordingly. This will make this class more focused (SRP adherent: https://en.wikipedia.org/wiki/Single-responsibility_principle)